siliconcompiler/tools/vpr/route.py

```diff
@@ -0,0 +1,84 @@
+import os
+import re
+import shutil
+
+from siliconcompiler.tools.vpr import vpr
+
+
+def setup(chip):
+    '''
+    Perform automated place and route with VPR
+    '''
+
+    tool = 'vpr'
+    step = chip.get('arg', 'step')
+    index = chip.get('arg', 'index')
+    task = chip._get_task(step, index)
+
+    chip.set('tool', tool, 'exe', tool, clobber=False)
+    chip.set('tool', tool, 'version', '0.0', clobber=False)
+
+    chip.set('tool', tool, 'task', task, 'threads', os.cpu_count(),
+             step=step, index=index, clobber=False)
+
+    # TO-DO: PRIOROTIZE the post-routing packing results?
+    design = chip.top()
+    chip.add('tool', tool, 'task', task, 'output', design + '.route', step=step, index=index)
+    chip.add('tool', tool, 'task', task, 'output', 'vpr_stdout.log', step=step, index=index)
```

Comment:
is this file an actual output? if not, it should not be recorded as an output

Reply:
Can't remember why I included the log file in the list of outputs; removed.